pkg/local_object_storage/metabase/metadata.go

```diff
@@ -534,8 +534,8 @@ nextPrimKey:
 		// collect attributes
 		collected := make([]string, len(attrs))
 		if len(attrs) > 0 {
+			var err error
 			if intPrimMatcher {
-				var err error
```

Comment:
Looks like belonging to the second commit, not the first one.

Reply:
yeah fixed